CONTRIBUTING.md

```diff
@@ -82,6 +82,22 @@ To fix all linting issues, run:
 $ pnpm lint:fix
 ```
 
+## Releases
+
+To release a new version follow these steps make sure that some prerequisites are met:
+
+- The smoke tests are passing.
+- There are changesets under `.changeset`. Without changesets there should be nothing to release.
+
+If this is the case, follow these steps:
+
+1. **Bump the version**: Execute the `bump` workflow from the `main` branch.
+   If you want to release a new major version, you need to enter the full major version as a precaution.
```

Comment:
```suggestion
   If you want to release a new major version, you need to enter the full major version as a precaution.
   Skip providing a version for minor version releases.
```
[question] How does it work for patch? Should we explicitly give the input?

Reply:
no, the way to you added it is correct for patch versions as well.